pkg/asset/installconfig/installconfig.go

```diff
@@ -88,16 +88,16 @@ func (a *InstallConfig) Generate(parents asset.Parents) error {
 		PullSecret: pullSecret.PullSecret,
 	}
 
-	numberOfMasters := int64(3)
-	numberOfWorkers := int64(3)
+	numberOfControlPlane := int64(3)
```

Comment:
There's only one control plane.  How about `numberOfControlPlaneMachines`?  And `numberOfComputeNodes` below?